native/src/testing/FormDataMock.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+type FormDataNameValuePairType = [string, string]
+type FormDataPartType = {
+  string: string
+} | {
+  uri: string,
+  name?: string,
+  type?: string
+}
+
+// See https://github.com/RealOrangeOne/react-native-mock/blob/master/src/Libraries/Network/FormData.js for reference
+class FormDataMock {
```

Comment:
FormData actually has some more fields: https://developer.mozilla.org/de/docs/Web/API/FormData

I think this should be a complete mock.

Reply:
Unnecessary work and complexity imo, we are not using anything of these fields.